docs/articles/documentation/test-api/assertions/assertion-api.md

```diff
@@ -0,0 +1,300 @@
+
+Asserts that `actual` is falsy.
+
+```js
+await t.expect('ok').notOk('this assertion will be failed');
+await t.expect(false).notOk('this assertion will be passed');
+```
+
+## contains
+
+```text
+await t.expect(actual).contains(expected, message, options);
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | Any type | A String or Array that should contain an `expected` value
+`expected`             | Any type | An expected value
+`message`&#160;*(optional)* | String   | An assertion message that will be displayed in the report if the test fails
+`options`&#160;*(optional)* | Object   | See [Options](index.md#assertion-options).
+
+Asserts that `actual` contains `expected`. `Actual` should be String or Array.
+
+```js
+await t.expect('foo bar').contains('bar', 'actual string contains expected');
+await t.expect([1, 2, 3]).contains(2, 'actual array contains expected');
+```
+
+## notContains
+
+```text
+await t.expect(actual).notContains(expected, message, options);
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | Any type | A String or Array that should not contain an `expected` value
+`expected`             | Any type | An expected value
+`message`&#160;*(optional)* | String   | An assertion message that will be displayed in the report if the test fails
+`options`&#160;*(optional)* | Object   | See [Options](index.md#assertion-options).
+
+Asserts that `actual` does not include `expected`. `Actual` should be String or Array.
+
+```js
+await t.expect('foo bar').notContains('baz', 'actual string do not contains expected');
+await t.expect([1, 2, 3]).notContains(4, 'actual array contains expected');
+```
+
+## typeOf
+
+```text
+await t.expect(actual).typeOf(typeName, message, options);
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | Any type | A comparison value
+`typeName`             | String | The expected type of an `actual` value
+`message`&#160;*(optional)* | String   | An assertion message that will be displayed in the report if the test fails
+`options`&#160;*(optional)* | Object   | See [Options](index.md#assertion-options).
+
+Asserts that type of `actual` is `typeName`. Type of `actual` will be determined as a constructor's name.
```

Comment:
I'm still not sure about the phrase ```will be determined as a constructor's name.``` Can i use `number`, `Number`, `NUMBER`? I can't understand it from docs.